source/B2BApi/Functions/EnqueueMessages/BRS_021/EnqueueHandler_Brs_021_Calculations_V1.cs

```diff
@@ -0,0 +1,80 @@
+using Microsoft.Azure.Functions.Worker.Http;
+using NodaTime.Extensions;
+
+namespace Energinet.DataHub.EDI.B2BApi.Functions.EnqueueMessages.BRS_021;
+
+public class EnqueueHandler_Brs_021_Calculations_V1(
+    IOutgoingMessagesClient outgoingMessagesClient,
+    IUnitOfWork unitOfWork)
+{
+    private readonly IOutgoingMessagesClient _outgoingMessagesClient = outgoingMessagesClient;
+    private readonly IUnitOfWork _unitOfWork = unitOfWork;
+
+    public async Task HandleAsync(HttpRequestData request, CancellationToken hostCancellationToken)
+    {
+        var requestBodyStream = request.Body;
+        using var reader = new StreamReader(requestBodyStream);
+        var requestBody = await reader.ReadToEndAsync(hostCancellationToken).ConfigureAwait(false);
+        var measurements = JsonSerializer.Deserialize<EnqueueCalculatedMeasurementsHttpV1>(requestBody)!;
+
+        foreach (var receiversWithMeasurements in measurements.Data)
+        {
+            foreach (var receiver in receiversWithMeasurements.Receivers)
+            {
+                var energyObservations = receiversWithMeasurements.Measurements
+                    .Select(x =>
+                        new EnergyObservationDto(
+                            Position: x.Position,
+                            Quantity: x.EnergyQuantity,
+                            Quality: Quality.FromName(x.QuantityQuality.Name)))
+                    .ToList();
+
+                var acceptedForwardMeteredDataMessageDto = new CalculatedMeasurementsMessageDto(
+                    eventId: EventId.From(Guid.CreateVersion7()),
+                    externalId: new ExternalId(measurements.TransactionId),
+                    receiver: new Actor(
+                        ActorNumber.Create(receiver.ActorNumber),
+                        ActorRole.FromName(receiver.ActorRole.Name)),
+                    businessReason: BusinessReason.PeriodicMetering,
+                    gridAreaCode: receiversWithMeasurements.GridAreaCode,
+                    series: new ForwardMeasurementsMessageSeriesDto(
+                        TransactionId: TransactionId.From(measurements.TransactionId.ToString()),
+                        MarketEvaluationPointNumber: measurements.MeteringPointId,
+                        MarketEvaluationPointType: MeteringPointType.FromName(measurements.MeteringPointType.Name),
+                        OriginalTransactionIdReferenceId: null,
+                        Product: "8716867000030",
+                        QuantityMeasureUnit: MeasurementUnit.FromName(measurements.MeasureUnit.Name),
+                        RegistrationDateTime: receiversWithMeasurements.RegistrationDateTime.ToInstant(),
+                        Resolution: Resolution.FromName(measurements.Resolution.Name),
+                        StartedDateTime: receiversWithMeasurements.StartDateTime.ToInstant(),
+                        EndedDateTime: receiversWithMeasurements.EndDateTime.ToInstant(),
```

Comment:
```suggestion
                        StartedDateTime: receiversWithMeasurements.StartDateTime.ToInstant(),
                        EndedDateTime: receiversWithMeasurements.EndDateTime.ToInstant(),
```

I would prefer `Start`/`End` instead of using the past tense `Started`/`Ended`.